spec/requests/content_pages_spec.rb

```diff
@@ -46,35 +48,32 @@
   end
 
   describe 'GET /modules/:training_module_id/content-pages/1-3-4' do
-    before do
-      get training_module_content_page_path(:alpha, '1-3-4')
-    end
-
     context 'when module is not completed' do
```

Comment:
and add a `before` block in this `context`

Reply:
shall we talk about it?  that is they way I had it before and it didn't work.